x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/node_attrs_details/node_attrs_details.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyout,
+  EuiTitle,
+  EuiInMemoryTable,
+  EuiSpacer,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiPortal,
+} from '@elastic/eui';
+import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
+
+
+export class NodeAttrsDetailsUi extends PureComponent {
+  static propTypes = {
+    fetchNodeDetails: PropTypes.func.isRequired,
+    close: PropTypes.func.isRequired,
+
+    details: PropTypes.array,
+    selectedNodeAttrs: PropTypes.string.isRequired,
+    allocationRules: PropTypes.object,
+  };
+
+  componentWillMount() {
+    this.props.fetchNodeDetails(this.props.selectedNodeAttrs);
+  }
+
+  render() {
+    const { selectedNodeAttrs, allocationRules, details, close, intl } = this.props;
+
+    return (
+      <EuiPortal>
+        <EuiFlyout ownFocus onClose={close}>
+          <EuiFlyoutBody>
+            <EuiTitle>
+              <h2>
+                <FormattedMessage
+                  id="xpack.indexLifecycleMgmt.nodeAttrDetails.title"
+                  defaultMessage="Nodes that contain the attribute: {selectedNodeAttrs}"
+                  values={{ selectedNodeAttrs }}
+                />
+              </h2>
+            </EuiTitle>
+            <EuiSpacer size="s" />
+            {allocationRules ? (
+              <Fragment>
+                <EuiCallOut
+                  style={{ marginTop: '1rem' }}
```

Comment:
Can you add another `EuiSpacer` above the call out instead of using this style?

Reply:
done